src/org/omegat/Bundle.properties

```diff
@@ -2189,6 +2189,7 @@ SCW_RUN_SCRIPT=&Run
 SCW_CANCEL_SCRIPT=Cancel
 SCW_CANCEL_BUTTON_TOOLTIP=Cancel only works on scripts that explicitly check for cancellation.
 SCW_RUNNING_SCRIPT=The script "{0}" is running...
+SCW_RUNNING_EDITOR_SCRIPT=Running the script of scripting editor...
```

Comment:
Please review a message.

In previous, the message for editor default script running is recorded as `The script "<scripts>" is running...`
Here propose to change it `Running the script of scripting editor...`